puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/plugins/bastion.yaml

```diff
@@ -8,6 +8,7 @@
           basedir: 'plugin'
           branches:
             - '${branch}'
+          refspec: '+refs/heads/*:refs/remotes/origin/* +refs/pull/*:refs/remotes/origin/pr/*'
```

Comment:
In other ghprb jobs we use `refspec: '+refs/pull/${ghprbPullId}/*:refs/remotes/upstream/pr/${ghprbPullId}/*'` to only pull in the PR we're testing.

Reply:
`test_foreman_javascript` isn't a GHPRB job itself, but it can sometimes get kicked by `test_bastion_pull_request` which is and it needs to know how to get at a PR. Here I am telling git about the standard refspec (`+refs/heads/*:refs/remotes/origin/*`) as well as the refspec for grabbing pull-requests (`+refs/pull/*:refs/remotes/origin/pr/*`).